astro.config.mjs

```diff
@@ -13,5 +13,28 @@ export default defineConfig({
   vite: {
     plugins: [yaml()]
   },
-  base: "/bioimage-archive"
+  base: "/bioimage-archive",
+  redirects: {
```

Comment:
i decided to be exhaustive and list all redirects i caused - but many of these pages dont really exist on the current website, so i don't know that we need to be worried about them.